gluten-core/src/main/scala/org/apache/gluten/softaffinity/SoftAffinityManager.scala

```diff
@@ -88,6 +90,13 @@ abstract class AffinityManager extends LogLevelUtil with Logging {
         }
       })
 
+  private var hashRing: Option[ConsistentHash[ExecutorNode]] = None
```

Comment:
Can we make it a `val`? Likely it's only set once.

Reply:
thanks, addressed.